### DISCUSSION TITLE
Create user personas for the Hubverse

### DISCUSSION THREAD

**Discussion by zkamvar** [hubverse-org/.github#27] [General]
(N.B. this is very much a duplicate of https://github.com/hubverse-org/hubUtils/discussions/11)

One of the things that strikes me as I walk through the documentation is that people interact with the hubs in very different ways depending on their role. Having a clear description of what these roles entail would be really helpful to structure the documentation in a way that meets the needs of the people who assume these roles so that (e.g.) a modeller does not have to sift through hub admin documentation to understand how to format their output. 

From my initial discussion with @bsweger, she identified four roles for a hub:

1. Hub Administrators
2. Modellers
3. Analysts
4. Developers

There may be more, but this is what I'm starting with. I would like to gather some [user personas](https://www.qualtrics.com/experience-management/research/user-personas/) for these roles. **Please help me by sharing what you know about these roles.**

Importantly, user personas are not just _what_ the roles are doing, but also what the people in these roles want to accomplish. Perhaps most importantly, they include factors outside of the role responsibilities that may influence how much time or energy someone may have to spend on their role. 

As an example of user personas, at my previous work, we worked for a community of volunteers who were teaching researchers to work with code and data. These are the personas for learners in our workshops: https://carpentries.org/audience/

**Comment by annakrystalli** [comment]
There was a bit of an effort to create these formally at the start but it didn't get very far and was superceded by the User Roles documentation.

Here's as far as we got: https://docs.google.com/document/d/17uqJ61lcVErfFm7uzWqMMopbLLAO6GSSR46KlV5zJ24/edit

**Reply by zkamvar** [reply]
Ah! I see the old discussion! https://github.com/hubverse-org/hubUtils/discussions/11
I tried searching in the org discussions, but I guess GitHub doesn't like to show discussions for an organization that exist in a specific repo 🤷🏽 

> superceded by the User Roles documentation.

I cannot find this documentation anywhere. Is there a link?

**Reply by annakrystalli** [reply]
Sorry for the confusion, I meant the section in the paper that @mmkerr already shared in slack! 😬

So there isn't really anything formal.

Definitely think this is a worthy exercise though!

**Reply by annakrystalli** [reply]
slide 20 in the following slide deck might also be useful  
https://docs.google.com/presentation/d/1518iadkaPnFRnzblQAdnrsMbORhdTJP7Y_kW7Gu3Uyg/edit
